ll.c

```diff
@@ -432,6 +432,25 @@ void stfs_ll_op_statfs(fuse_req_t req, fuse_ino_t ino) {
 	}
 }
 
+void sqfs_ll_op_init(void *userdata, struct fuse_conn_info *conn) {
+	sqfs_ll *ll = userdata;
+
+	if (!ll->fs.notify_pipe) {
+		return;
+	}
+
+	pid_t pid = fork();
```

Comment:
This logic is a bit repetitive between the hl/ll cases. Maybe we can add a common function in fuseprivate?